lib/widgets/atoms/explore_filter.dart

```diff
@@ -1,18 +1,20 @@
+import 'dart:js_interop';
+
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import '../../constants/app_constants.dart';
 
 class ExploreFilter extends StatelessWidget {
   final UserType userType;
-  final bool userFiltersSelected;
+  // final bool userFiltersSelected;
```

Comment:
If this is not needed then it is best to delete it to avoid cluttering up the code.

Reply:
Absolutely, removed.